config/manifests/bases/jaeger-operator.clusterserviceversion.yaml

```diff
@@ -124,7 +124,7 @@ spec:
   maturity: alpha
   provider:
     name: CNCF
-  replaces: jaeger-operator.v1.30.0
+  replaces: jaeger-operator.v1.32.0
```

Comment:
shouldn't it be v1.31.0 here? 
(`OPERATOR_VERSION=1.32.0 make prepare-release` was called)

Reply:
yes, this should be 1.31.0